src/Microsoft.AspNetCore.ResponseCaching/Internal/ResponseCacheContext.cs

```diff
@@ -37,6 +40,8 @@ public class ResponseCacheContext
 
         public CachedVaryByRules CachedVaryByRules { get; internal set; }
 
+        internal ILogger Logger { get; }
```

Comment:
talked with @JunTaoLuo and @Tratcher, I bought the explanation. keep it internal then.